jekyll/_cci2/using-linuxvm.md

```diff
@@ -1,29 +1,24 @@
 ---
 layout: classic-docs
 title: "Using the Linux VM execution environment"
-description: "Learn how to configure a your jobs to run in the Linux VM execution environment."
+description: "Learn how to configure a your jobs to run in the Linux VM execution environment using the machine executor."
 version:
 - Cloud
 - Server v3.x
 - Server v2.x
 ---
 
-The `machine` option runs your jobs in a dedicated, ephemeral VM that has the following specifications:
+Your can run your jobs in the linux VM (virtual machine) execution environment by using the machine executor and specifying a Linux image. Using the machine executor runs your jobs in a dedicated, ephemeral VM.
 
-{% include snippets/machine-resource-table.md %}
-
-Using the `machine` executor gives your application full access to OS resources and provides you with full control over the job environment. This control can be useful in situations where you need full access to the network stack; for example, to listen on a network interface, or to modify the system with `sysctl` commands. To find out about migrating a project from using the Docker executor to using `machine`, see the [Executor Migration from Docker to Machine]({{ site.baseurl }}/2.0/docker-to-machine) document.
+Using the `machine` executor gives your application full access to OS resources and provides you with full control over the job environment. This control can be useful in situations where you need full access to the network stack; for example, to listen on a network interface, or to modify the system with `sysctl` commands.
```

Comment:
Do we still want to style machine here with backticks?

Reply:
Nope thank you!